AEPLifecycle/Tests/FunctionalTests/LifecycleV2FunctionalTests.swift

```diff
@@ -133,23 +136,28 @@ class LifecycleV2FunctionalTests: XCTestCase {
 
         // test
         // appplication launch install hit
-        mockRuntime.simulateComingEvents(createStartEvent())
+        let startEvent = createStartEvent()
+        mockRuntime.simulateComingEvents(startEvent)
         waitForProcessing(interval: 1.1) // app close after 1 sec
         // application close
-        mockRuntime.simulateComingEvents(createPauseEvent())
+        let pauseEvent = createPauseEvent()
+        mockRuntime.simulateComingEvents(pauseEvent)
         waitForProcessing(interval: Self.PAUSE_UPDATE_TIMEOUT)
 
         // verify
         XCTAssertEqual(2, mockRuntime.dispatchedEvents.count) //application launch, application close
 
         // event data
+        let dispatchedStartEvent = mockRuntime.dispatchedEvents[0]
         let dispatchedCloseEvent = mockRuntime.dispatchedEvents[1]
         let xdm = dispatchedCloseEvent.data?["xdm"] as? [String:Any] ?? [:]
         XCTAssertEqual("Application Close (Background)", dispatchedCloseEvent.name)
         XCTAssertEqual(EventType.lifecycle, dispatchedCloseEvent.type)
         XCTAssertEqual(EventSource.applicationClose, dispatchedCloseEvent.source)
         XCTAssertNotNil(xdm["timestamp"] as? String)
         XCTAssertTrue(NSDictionary(dictionary: xdm["application"] as? [String : Any] ?? [:]).isEqual(to: expectedApplicationInfo))
+        XCTAssertEqual(startEvent.id, dispatchedStartEvent.parentID)
+        XCTAssertEqual(pauseEvent.id, dispatchedCloseEvent.parentID)
     }
 
     func testLifecycleV2_appUpgrade() {
```

Comment:
Can you add checks for parent Id in the following tests?
testLifecycleV2_appCrash
testLifecycleV2_appCrash_closeTSMissing